packages/wp-now/src/wp-playground-wordpress/is-valid-wordpress-version.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Checks if the given version string is a valid WordPress version.
+ *
+ * The Regex is based on the releases on https://wordpress.org/download/releases/#betas
+ * The version string can be one of the following formats:
+ * - "latest"
+ * - "x.y" (x and y are integers) i.e. "6.2"
+ * - "x.y.z" (x, y and z are integers) i.e. "6.2.1"
+ * - "x.y.z-betaN" (N is an integer) i.e. "6.2.1-beta1"
+ * - "x.y.z-RCN" (N is an integer) i.e. "6.2-RC1"
```

Comment:
💯 love this - it's so clear what we're aiming for

Reply:
though I think `e.g.` is more fitting than `i.e.` whereas `e.g.` communicates "this is one example of what we might find" `i.e.` communicates, "in other words, _this_"

Reply:
Changed here! https://github.com/WordPress/wordpress-playground/pull/252/commits/be3a15539d5bed1cd18253705b7ff4e89c869650
Thank you!